packages/react/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -22,7 +22,10 @@ export const SegmentedControl = React.forwardRef((props, forwardedRef) => {
 
   const state = useRadioGroupState(props);
 
-  const { radioGroupProps } = useRadioGroup(props, state);
+  const { radioGroupProps } = useRadioGroup({
+    'aria-label': props?.value || 'Radio Group',
+    ...props
+  }, state);
```

Comment:
Wouldn't this just be indicative that whoever is implementing this didn't specify an `aria-label` on the segmented control?

i.e.
```
<Segmented control aria-label="Something" />
```

in the implementation?
```